src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,17 @@
-export const TodoList = () => {};
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = {
```

Comment:
You've already started nicely to export typing to separate files, so it would be cool to continue here as well, and export the type for the `TodoList` component to the `Todo.ts` file and give this type a more convenient name (For example: `TodoListType`).